src/overlay.jsx

```diff
@@ -134,6 +131,16 @@ const Overlay = React.createClass({
     if (!this.props.autoLockScrolling) this._allowScrolling();
```

Comment:
`componentWillUnmount` calls _allowScrolling, had a think about combining, but think its best to keep this separate.  Does that sound ok?